### ISSUE TITLE
Issue with Jekyll liquid templates and raw tags

### ISSUE THREAD

**Issue by cfjedimaster** [11ty/eleventy#737]
**Describe the bug**
I copied some posts I had written for my Jekyll blog and noticed that Eleventy's liquid parser did not like this:

```
{% raw %}
stuff
{% endraw %}
```
I use this when I blog about Vue and need to have it ignore Vue-related stuff for my post. When I ran Eleventy, it told me that the `raw` tag wasn't closed. Is this something I can configure Eleventy's Liquid to understand or some mistake on my part?

Note - I wasn't using `foo.liquid` but `foo.md`.

**To Reproduce**
Try to use unparsed Liquid code in a Markdown file by using raw/endraw wrappers.

**Expected behavior**
Liquid should not parse the code between them.

**Comment by zachleat** [comment]
Hmm both of these pass in our tests:

![image](https://user-images.githubusercontent.com/39355/68973928-1457e100-07b5-11ea-8018-5aa0a0c944e1.png)

Can you provide a more detailed reduced test case? What’s your markdown template language?

**Comment by cfjedimaster** [comment]
Well shoot, now I can't reproduce it. Maybe a gremlin or something else?

**Comment by cfjedimaster** [comment]
FYI I just reproduced it - but I'm boarding a plane for hours. My first test (today) was with a few of my blog posts. I just tried with *all* 9k of my blog posts and reproduced it. I'll narrow it down later today.

**Comment by cfjedimaster** [comment]
Ahah, I figured it out. It was *not* with a simple `{% raw %} stuff {% endraw %}` template as I thought. It was an earlier syntax I used on my blog and SUPER weird which is why I was so happy when I found the 'proper' way to escape. Anyway, this is what I was doing, it works on my blog (Jekyll), not Eleventy, which is fine.

```
      {% raw %}{{phone.name}{% endraw %}}
```

Note the weird closing brackets. It looks and feels wrong, but I swear this worked. Anyway, thank you!
